File: src/main/java/me/centralhardware/znatoki/telegram/statistic/entity/Services.java
package me.centralhardware.znatoki.telegram.statistic.entity;

import lombok.Getter;
import lombok.Setter;
import me.centralhardware.znatoki.telegram.statistic.eav.Property;

import java.util.List;
import java.util.UUID;

@Setter
@Getter
public class Services {

    private Long id;
    private String key;
    private String name;
    private UUID orgId;
    private List<Property> properties;

}

File: src/main/kotlin/me/centralhardware/znatoki/telegram/statistic/entity/Services.kt
package me.centralhardware.znatoki.telegram.statistic.entity

import kotliquery.Row

data class Services(
    var id: Long,
    var name: String,
    var allowMultiplyClients: Boolean
)

fun Row.parseServices() = Services(long("id"), string("name"), boolean("allow_multiply_clients"))
